### PR TITLE
Frontend stock data update

### PR DESCRIPTION
**Pull request by LuchoStoymirov**
image
image

### CODE REVIEW

File: TypeScript/AI_Finance/src/Components/StockData/StockData.tsx [hunk 1]
```diff
@@ -2,29 +2,43 @@ import style from "./StockData.module.css";
...
-  //   })();
-  // }, []);  
+
+  useEffect(() => {
+    (async () => {
+      try {
+        setData(await apiService.getStockData());
+      } catch (error) {
+        // eslint-disable-next-line no-console
+        console.log(error);
```
Comment (blankfactor-eleonora-yaneva) [inline TypeScript/AI_Finance/src/Components/StockData/StockData.tsx:17]: does the console log need to stay?

File: TypeScript/AI_Finance/src/Components/StockData/StockData.tsx [hunk 2]
```diff
@@ -2,29 +2,43 @@ import style from "./StockData.module.css";
...
+    })();
+  }, []);
+
+  const getStocks = () => {
+    (async () => {
+      try {
+        setData(await apiService.getStockData());
+      } catch (error) {
+        // eslint-disable-next-line no-console
+        console.log(error);
```
Comment (blankfactor-eleonora-yaneva) [inline TypeScript/AI_Finance/src/Components/StockData/StockData.tsx:29]: does the console log need to stay?

File: TypeScript/AI_Finance/src/Components/StockData/StockData.tsx [hunk 3]
```diff
@@ -1,10 +1,54 @@
...
+    item: {
+      imageUrl: string;
+      name: string;
+      price: string;
+      marketCap: string;
+      change: string;
+    },
+    id: number
+  ) => {
+    console.log(data);
```
Comment (blankfactor-eleonora-yaneva) [inline TypeScript/AI_Finance/src/Components/StockData/StockData.tsx:41]: remove the console.log

**Comment by LuchoStoymirov** [comment]
image
image

**Pull request merged**